src/ServiceBus/ServiceBus/Cmdlets/AuthorizationRule/NewAzureServiceBusAuthorizationRuleSASToken.cs

```diff
@@ -0,0 +1,111 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using System.Collections.Generic;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.ServiceBus.Models;
+using System;
+using System.Security.Cryptography;
+using System.Web;
+using System.Globalization;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.ServiceBus.Commands
+{
+    /// <summary>
+    /// 'New-AzEventHubAuthorizationRuleSASToken' Cmdlet creates a new AuthorizationRule
+    /// </summary>
+    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ServiceBusAuthorizationRuleSASToken", DefaultParameterSetName = NamespaceAuthoRuleParameterSet, SupportsShouldProcess = true), OutputType(typeof(string))]
```

Comment:
@v-Ajnava cmdlets should not return strings -- we should have a separate object that encapsulates the SAS token (and any additional information we would want to include) and return that instead

Reply:
Resolved